vllm/model_executor/models/clip.py

```diff
@@ -407,12 +416,12 @@ def __init__(
         # NOTE: This typo of "layrnorm" is not fixed on purpose to match
         # the original transformers code and name of the model weights.
         self.pre_layrnorm = nn.LayerNorm(embed_dim, eps=config.layer_norm_eps)
+
         self.encoder = CLIPEncoder(
             config=config,
             quant_config=quant_config,
             num_hidden_layers_override=num_hidden_layers_override,
-            prefix=f"{prefix}.encoder",
```

Comment:
Seems like an unnecessary formatting change.

Reply:
Yup! I put it back the way it was, accidentally deleted some trailing commas + auto formatted in some places 😄